app/models/setting/provisioning.rb

```diff
@@ -75,6 +75,7 @@ def self.default_settings
       set('default_pxe_item_global', N_("Default PXE menu item in global template - 'local', 'discovery' or custom, use blank for template default"), nil, N_("Default PXE global template entry")),
       set('default_pxe_item_local', N_("Default PXE menu item in local template - 'local', 'local_chain_hd0' or custom, use blank for template default"), nil, N_("Default PXE local template entry")),
       set('intermediate_ipxe_script', N_('Intermediate iPXE script for unattended installations'), 'iPXE intermediate script', N_('iPXE intermediate script'), nil, { :collection => proc { Hash[ProvisioningTemplate.unscoped.of_kind(:iPXE).map { |tmpl| [tmpl.name, tmpl.name] }] } }),
+      set('templates_rendering_status_refresh_interval', N_('The refreshing interval for templates rendering status (in minutes)'), 5, N_('Templates rendering status refresh interval')),
```

Comment:
This can be very costly, imagine a deployment with 50000 hosts, this can quickly get longer than 5 minutes. I suggest 15 at least for the default value, if not an hour.